storage/rocksdb/ha_rocksdb.cc

```diff
@@ -10897,9 +10897,9 @@ struct rocksdb_status_counters_t {
   uint64_t bloom_filter_prefix_checked;
   uint64_t bloom_filter_prefix_useful;
   uint64_t number_reseeks_iteration;
-  uint64_t getupdatessince_calls;
-  uint64_t block_cachecompressed_miss;
-  uint64_t block_cachecompressed_hit;
+  uint64_t get_updates_since_calls;
```

Comment:
getupdatessince was referred to as "GetUpdatesSince" in the MyRocks status var doc PR - is that an actual name in RocksDB? In which case it should be left as-is. And the request to fix cachecompressed was based on a premise that there is no any e.g. "CacheCompressed" in RocksDB?

Reply:
All of these values are extracted from rocksdb via an API and identifiers which are 'mapped' into MySQL status variables. The identifiers for these are GET_UPDATES_SINCE_CALLS, BLOCK_CACHE_COMPRESSED_MISS, BLOCK_CACHE_COMPRESSED_HIT.

The text for the documentation was scraped from comments within the code and so could easily have been inconsistent.